tensorflow/lite/micro/memory_arena_threshold_test.cc

```diff
@@ -87,6 +85,12 @@ uint8_t test_conv_tensor_arena[kTestConvModelArenaSize];
 constexpr int kTestConvModelTensorCount = 15;
 constexpr int kTestConvModelNodeAndRegistrationCount = 7;
 
+#if defined(USE_TFLM_COMPRESSION)
+constexpr int kKeywordModelPersistentBufferDataSize = 920;
```

Comment:
Is the buffer size expected to grow by this much, even when the model does not utilize compression? I'm a bit curious where the overhead comes from.

Reply:
I believe it comes from the new FCOpData for compression scratch indices. It's behind the compression ifdef. So if you enable compression define, but don't use compression you will see the overhead, but if you don't enable compression build, it should remain same as before. Needs to remain persistent because you need the data at different times (prepare/eval).

Reply:
@suleshahid Correct.